src/test/unit/snykOss/services/vulnerabilityCount/parsers/moduleParserProvider.test.ts

```diff
@@ -10,7 +10,7 @@ suite('OSS ModuleParserProvider', () => {
   });
 
   test('Undefined instance is returned for non-supported language', () => {
-    const parser = ModuleParserProvider.getInstance(SupportedLanguage.HTML);
+    const parser = ModuleParserProvider.getInstance(99);
```

Comment:
took me a while to figure out that enum values are transpiled to integers :).
does `-1` suit better to show non-supported language?

Reply:
-1 would suit definitely better 👍 thanks for pointing it out, fresh eyes 👀